### ISSUE TITLE
Instagram unshown

### ISSUE THREAD

**Issue by ghost** [potomak/jquery-instagram#59]
Hello, This is Ryo, web director from Japan.

I have a question about Instagram plugin.
The website we built have not shown Instagram from yesterday.
could be cause of ：https://www.instagram.com/developer/

could you clear this problem??

thank you.

**Comment by potomak** [comment]
Yes, it seems like the use case that this plugin tries to cover is not supported anymore by the new API policies.

I was trying to submit the example app used at http://potomak.github.io/jquery-instagram/ for review and this is the result:

> This use case is not supported. We do not approve the public_content permission for one-off projects such as displaying hashtag based content on your website. As alternative solution, you can show your own Instagram content, or find a company that offers this type of service (content discover, moderation, and display).

![screen shot 2016-06-03 at 9 48 34 am](https://cloud.githubusercontent.com/assets/153842/15780588/6eff3580-2970-11e6-8012-0380622fe141.png)

More info at https://www.instagram.com/developer/review/.

**Comment by xwisnux** [comment]
Use **userId** and **accessToken** instead of clientId, this works for me. You can get your acces token here: http://instagramwordpress.rafsegat.com/docs/get-access-token/

`$('.instagram.tag').on('didLoadInstagram', didLoadInstagram);
    $('.instagram.tag').instagram({
        hash: 'YOUR_HASH',
        count: 12,
        userId: USER_ID,
        accessToken: ACCESS_TOKEN
    });`

**Comment by potomak** [comment]
@xwisnux you're correct, thanks for pointing this out.

I must say I liked it better before when you didn't need to share an access token publicly to use the Instagram API.

**Comment by xwisnux** [comment]
@potomak Correct, but it seems that is our only best shot at the moment. Hopefully there will be another way than showing our access token.

**Comment by ghost** [comment]
@potomak @xwisnux 
Thank you for telling it me! 
It worked from using new access toke yesterday but it is unshown again....
I would search another way. though thank you for your reply.
